### ISSUE TITLE
loss.backward() attributeerror 'float' object has no attribute 'backward'

### ISSUE THREAD

**Issue by alexwg0819** [swz30/MPRNet#108]
loss.backward() attributeerror 'float' object has no attribute 'backward' When i run train.py, plx help

**Comment by adityac8** [comment]
Hi 

This might help
https://discuss.pytorch.org/t/getting-error-float-object-has-no-attribute-backward/98597/2

Thanks
